mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -486,7 +486,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
-	METHOD method_43688 asMessageSender ()Lnet/minecraft/class_7436;
+	METHOD method_43688 asMessageSourceProfile ()Lnet/minecraft/class_7436;
```

Comment:
Maybe this could be `getMessageSourceProfile`?

Reply:
yeah, "as" prefix made sense before, but unsure here. Note that this isn't a getter.

Reply:
It could also be "to" or "create", but I think "get" is the simplest option.